chart.js

```diff
@@ -152,10 +156,16 @@ function createChart(allData, year) {
         'TX-BUSY-2048chunksize-64iosize-FNDIS',
         'TX-BUSY-2048chunksize-1514iosize-FNDIS'
     ]
-    createChartwithData(allData, year, "XDPMP-NATIVE", names, true, false)
-    createChartwithData(allData, year, "XDPMP-GENERIC", names, true, false)
-    createChartwithData(allData, year, "XDPMP-RIO", names, true, false)
-    createChartwithData(allData, year, "XDPMP-WINSOCK", names, true, false)
+    var colors = [
```

Comment:
I wonder if there are recommended color sets for accessibility, e.g. to reduce ambiguity for color blindness? Hovering still removes ambiguity of course.

Reply:
Prob avoiding red+green is the number one tip, but considering how small the audience is for this dashboard, I'd say this is overdesign (assuming nobody who is looking at it today is known to be red-green colorblind).

Slightly related, at first I was thinking I'd hash the testcase strings to select colors (lol), so we could avoid having a hardcoded list of colors and you wouldn't have to select a new color if you added a testcase; but it leads to some obvious new problems.